src/ui/pages/payment-entry-page.svelte

```diff
@@ -180,6 +181,12 @@
         signal,
       )
       .then((taxCalculation) => {
+        const event = createCheckoutPaymentTaxCalculationEvent({
```

Comment:
Should we add a comment indicating that we intentionally track these events even if the calculation is aborted?